### PR TITLE
Migrate to slf4j-simple logger in tests

### LINKED ISSUE #819: Exhaustive logging causes travis to kill the build

### PR DESCRIPTION
**Pull request by jimirocks**
Resolves #819

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: src/test/resources/simplelogger.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+org.slf4j.simpleLogger.logFile=System.out
+org.slf4j.simpleLogger.defaultLogLevel=debug
```
Comment (liry) [inline src/test/resources/simplelogger.properties]: you meant info?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: src/test/resources/simplelogger.properties (hunk 1 above)
Comment (jimirocks) [reply src/test/resources/simplelogger.properties]: yeah, let's doit rather that way

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-java-merge-compliance-check/111/ : SUCCESS in 26s

**Pull request merged**